builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -91,6 +91,14 @@ the entire validity period. It is suggested to use "truncate" for
 intermediate CAs and "permit" only for root CAs.`,
 		Default: "err",
 	}
+	fields["usage"] = &framework.FieldSchema{
+		Type: framework.TypeCommaStringSlice,
+		Description: `Comma-separated list (or string slice) of usages for
+this issuer; valid values are "read only", "issuing certificates", and
+"CRL signing". Multiple values may be specified. Read-only is implicit
+and always set.`,
+		Default: []string{"issuing certificates", "CRL signing"},
```

Comment:
nit: Not a huge fan with the mixed casing and spaces for values here. What would you think of using the following instead?

- read-only
- issuing-certificates
- crl-signing

Reply:
Well, CRL is an acronym and calling it "crl" felt weird if I was going to use spaces ;-) 

I'm fine with slugs, but note this gets used as both "identifiers" and "names for error messages" so that's why I went with the space approach (better error messages, worse names). Fewer people will modify the values than read the errors (since it is user-returnable with a bad-ish configuration).

Reply:
Fair enough, the poor command line users needing to escape the values now ;)

Reply:
Eh, that's a good point, I'll change them.